pydrad/util/constants.py

```diff
@@ -0,0 +1,35 @@
+"""
+Constants used throughout `pydrad`
+"""
+from astropy.constants import Constant
+
+__all__ = [
+    'gamma',
+    'm_avg_ion',
+]
+
+
+gamma = Constant(
+    abbrev='gamma',
+    name='heat capacity ratio of a monatomic ideal gas',
+    value=5/3,
+    unit='',
+    uncertainty=0,
+    reference='https://en.wikipedia.org/wiki/Heat_capacity_ratio',
+    system=None,
+)
+
+
+# NOTE: This is pulled directly from the HYDRAD source code
+# https://github.com/rice-solar-physics/HYDRAD/blob/master/Resources/source/constants.h
+# and is the average ion mass for a H-He plasma computed by weighting the H and He ion
+# masses with a particular set of abundances.
+m_avg_ion = Constant(
+    abbrev='m_avg_ion',
+    name='average ion mass for a H-He plasma',
+    value=2.171e-24,
+    unit='g',
+    uncertainty=0,
```

Comment:
There's definitely an uncertainty, but I don't know what it is!  

The uncertainty from the abundance set (likely Feldman) + the uncertainty of the hydrogen and helium masses should propagate.  

Perhaps we should make an issue to figure this out?  I do like that this has the option of including it!